plugins/modules/proxysql_replication_hostgroups.py

```diff
@@ -33,6 +33,12 @@
     description:
       - Text field that can be used for any purposes defined by the user.
     type: str
+  check_type:
+    description:
+      - Which check type to use when detecting that the node is a slave.
```

Comment:
```suggestion
      - Which check type to use when detecting that the node is a standby.
```
just a suggestion. or we could use `replica` alternatively

Reply:
Done in 936071b106cbe536211938bec813bc0d8e89b703

Reply:
As for the tests - will do that later.